eogrow/core/area/batch.py

```diff
@@ -138,6 +141,152 @@ def sort_key_function(values: Tuple[Any, dict]) -> Tuple[str, int, int]:
 
         return AreaManager._to_dataframe_grid(bbox_list, info_list, info_columns)
 
+    def transform_grid(self, target_area_manager: AreaManager) -> List[GridTransformation]:
+        """Calculates how grid should be transformed into another area manager."""
+        self._check_target_compatibility(target_area_manager)
+        target_area_manager = cast(BatchAreaManager, target_area_manager)
+
+        source_grid = self.get_grid(add_bbox_column=True)
+        source_grid = _ensure_split_columns(source_grid)
+        target_grid = self._get_target_grid(target_area_manager, source_grid)
+
+        source_shape, target_shape = zip(*map(reduce_to_coprime, self.subsplit, target_area_manager.subsplit))
+        source_shape = cast(Tuple[int, int], source_shape)
+        target_shape = cast(Tuple[int, int], target_shape)
```

Comment:
Nice find, I couldn't think of this option. :bowing_man: 

I was also thinking about renaming "shape" but I couldn't find any better word. Because these numbers actually represent a shape of a grid of bboxes.